django/views/templates/technical_500.html

```diff
@@ -13,7 +13,7 @@
     body>div { border-bottom:1px solid #ddd; }
     h1 { font-weight:normal; }
     h2 { margin-bottom:.8em; }
-    h2 span { font-size:80%; color:#666; font-weight:normal; }
+    h2 span { font-size:80%; color:#4f4f4f; font-weight:normal; }
```

Comment:
I'm not sure this color is used. `span.commands a:link` seems to take precedence.